carbon-design-system/carbon-for-ibm-dotcom
# fix(cloud-masthead): address choppy transitions on mobile menu

### Related Ticket(s)

GH Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7089
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2182

### Description

- This PR forces GPU on elements with large compositing layers that are causing choppy and unsmooth animations.
- This is a test/draft PR to see if we can eliminate the choppy transitions on the WC test application.

### Testing Instruction

- Pull down the changes of this PR in a test application preview and test the animations on a simulator or devices to see if the choppy transition is still persistent within the mobile dropdown.

Test the Cloud Masthead Container:
```
<dds-cloud-masthead-container
  platform="Cloud"
  auth-method="API"
  has-contact=false
  data-endpoint="/common/carbon-for-ibm-dotcom/translations/cloud-masthead"
></dds-cloud-masthead-container>
```
